assets/components/contour/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v2
+name: contour
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number is incremented each time changes are made
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the upstream Contour
+# https://github.com/projectcontour/contour/releases
+appVersion: v1.3.0
```

Comment:
It must be semver, so no `v`.

Reply:
But we are following semver for `version` no? We are pointing to the upstream version verbatim. https://github.com/projectcontour/contour/releases/tag/v1.3.0

Reply:
No. SemVer spec says it must be X.Y.Z, not `vX.Y.Z`.

Reply:
But the docs say that it is okay to not have appVersion to be semver.

> appVersion: The version of the app that this contains (optional). This needn't be SemVer.

src: https://helm.sh/docs/topics/charts/

Reply:
oh :facepalm: I misread :smile: All good then. Sorry about the confusion!